initfiles/componentfiles/configxml/buildsetCC.xml.in

```diff
@@ -44,6 +44,11 @@
              processName="DaliServerProcess"
              schema="dali.xsd"/>
    <BuildSet installSet="deploy_map.xml"
+             name="daliplugin"
```

Comment:
This is a generic plugin.  The first use is probably going to be for cassandra wu server. The name 'daliplugin' seems reasonable.  Based on naming provided here https://github.com/hpcc-systems/HPCC-Platform/pull/8439